includes/definitions/eaton-ats.yaml

```diff
@@ -0,0 +1,14 @@
+os: eaton-ats
+text: 'Eaton ATS'
+type: power
+icon: eaton
+mib_dir:
+    - eaton
+over:
+    - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.705.1
```

Comment:
This will detect devices as eaton-mgeups, you will need to just detect on sysDescr_regex like you've used below and exclude that same regex from the mgeups OS, use `sysDescr_regex_except.

Reply:
https://travis-ci.org/librenms/librenms/jobs/285547619